test/java_time_test.clj

```diff
@@ -857,3 +857,24 @@
            (is (some? (j/formatter :rfc-1123-date-time)))
            (finally
              (Locale/setDefault current-locale))))))))
+
+(deftest formatter-test
+  (testing "case-insensitive formatter"
+    (let [fmt (java-time/formatter "hh:mma" {:case :insensitive})]
+      (is (= (j/local-time 0 34 0 0)
+             (j/local-time fmt "00:34am")
+             (j/local-time fmt "00:34AM")))))
+
+  (let [java-version (System/getProperty "java.version")]
+    (if (and (not= java-version "11") (re-matches #"\d+" java-version))
```

Comment:
I'm having trouble understanding this. If the Java version string <> "11" and consists only of digits then "Java 13 and above treats AM as invalid" tests that "12:32AM" cannot be parsed with case-sensitive parser. If Java version = "11" or consists not only of digits (what then? Digits with separators? Something else?) - "12:34am" cannot be parsed with a case-sensitive parser. 

Can you please clarify the tests and maybe add links or inline the extended explanation?